tests/expand/st10a-out.jsonld

```diff
@@ -0,0 +1,14 @@
+[{
+  "@id": "http://example/s",
+  "http://example/p": [{
+    "@reifies": [{
+      "@id": "http://example/s1",
+      "http://example/p1": [{
+        "@reifies": [{
+          "@id": "http://example/s2",
+          "http://example/p2": [{"@id": "http://example/o2"}]
+        }]
+      }]
+    }]
+  }]
+}]
```

Comment:
if the change in the input was applied, this would become:
```suggestion
[{
  "@id": "http://example/s",
  "@reifies": [{
    "@id": "http://example/s1",
    "@reifies": [{
      "@id": "http://example/s2",
      "http://example/p2": [{"@id": "http://example/o2"}]
    }]
  }]
}]
```